packages/@uppy/dashboard/src/index.js

```diff
@@ -651,7 +645,7 @@ module.exports = class Dashboard extends Plugin {
   superFocusOnEachUpdate = () => {
     const isFocusInUppy = this.el.contains(document.activeElement)
     // When focus is lost on the page (== focus is on body for most browsers, or focus is null for IE11)
-    const isFocusNowhere = document.activeElement === document.querySelector('body') || document.activeElement === null
+    const isFocusNowhere = document.activeElement === document.body || document.activeElement === null
```

Comment:
I'm _pretty_ sure this warning is a bug in the compat lint and not actually accurate, but I guess it doesn't matter in this case :)

Reply:
Yeah, I was surprised too, but believed it blindly ;) `document.body` also seems like it would be a tiny-tiny-not-noticeable-anywhere faster 😂